src/Elm/Kernel/List.js

```diff
@@ -70,6 +70,23 @@ var _List_map5 = F6(function(f, vs, ws, xs, ys, zs)
 	return _List_fromArray(arr);
 });
 
+var _List_append = F2(_List_ap);
+
+function _List_ap(xs, ys)
```

Comment:
nitpick - I wonder about naming here. For instance cons uses this convention `var _List_cons = F2(_List_Cons);` (note same naming but different cases) https://github.com/elm/core/blob/4fbcb15dfcdb3038030a744b77c230b1e3665036/src/Elm/Kernel/List.js#L16

Reply:
`_List_Cons` is a constructor (the function returns a type). `_List_cons` is a function, which calls the constructor. You’ll see it defined in the `List` module as a function.